common-docs/multiplayer.md

```diff
@@ -0,0 +1,47 @@
+# Multiplayer
+
+### ~ alert
+
+#### Supported editors
+
+The online multiplayer game feature is only available for [Microsoft MakeCode Arcade](https://arcade.makecode.com).
+
+### ~
+
+A multiplayer game is one where **2** or more (up to a maximum of **4**) players can play the same game together at the same time. A game session is started by a _host_ player and other players _join_ the game online with MakeCode's multiplayer game system. If a game is programmed using multiplayer controls, the game creator can choose to host a multiplayer game session and share the game's join code with other players.
+
+## Host a game #host-game
+
+You host a multiplayer game by using the **Share** button at the top of the editor screen.
+
+Choose **Host a multiplayer game** instead of creating a share link to your project.
+
+Once you choose to host your game, a new window opens with your game's join code. You share this code with the other players so that they can join your game. You invite other players to join your game in one of three ways:
+
+* Give them the six-digit game code and when they navigate to the [MakeCode Game Lobby](https://aka.ms/a9b), or https://aka.ms/a9b, they can enter the code to join.
```

Comment:
```suggestion
* Give them the six-digit game code and when they navigate to the [MakeCode Game Lobby](https://aka.ms/a9), or https://aka.ms/a9, they can enter the code to join.
```
The `b` suffix points to `/beta`. We want to route to the live endpoint, so just `aka.ms/a9` ( no `b`).